todo_app/api/migrations/0005_auto_20210216_1125.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.6 on 2021-02-16 18:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0004_auto_20210215_1125'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='todo',
+            name='owner',
+            field=models.CharField(max_length=20, null=True),
+        ),
```

Comment:
This migration should be merged with the other migrations before that alter this field. 
We can talk more later about how to do this.